### DISCUSSION TITLE
Задание 7

### DISCUSSION THREAD

**Discussion by eprilutskaya** [anastasiaprakapchuk/AmastyLearningNewTechnology#22] [General]
https://github.com/anastasiaprakapchuk/AmastyLearningNewTechnology/blob/48033bd237f58fb8f18101343ea45c66e61f40db/AnastasiaModule/Controller/Cart/Add.php#L152
getFirstItem возвращает объект, поэтому правильнее будет сделать после этого getQty()
$blacklistCollection->getFirstItem()->getQty()

После if пустая строка не нужна.

https://github.com/anastasiaprakapchuk/AmastyLearningNewTechnology/blob/48033bd237f58fb8f18101343ea45c66e61f40db/AnastasiaModule/Controller/Cart/Add.php#L117

можно объединить со следующим if в elseif () {

Оценка 9

**Comment by anastasiaprakapchuk** [comment]
Сделала исправления в ветке task-7
